### ISSUE TITLE
508 Issue: Aria Labels for QA Data Tables (Configurations Tab)

### ISSUE THREAD

**Issue by vishnunavuluri** [US-EPA-CAMD/PI-1#4055]
**Issue:**
On an individual configurations tab, the data tables have an incorrect aria label aria-label="DataTable Aria Label Missing"

Update the aria-label to aria-label="Test Summary Data" etc depending on the QA Test Section

**Comment by aprematta** [comment]
Aria label is now correct, closing ticket
